### PR TITLE
Added nodeIntegration: true to webpreferences

### PR DESCRIPTION
**Pull request by ahallicks**
Fixes require being undefined in app.js (and therefore breaking EVERYTHING).

### CODE REVIEW

File: lib/notifier.js [hunk 1]
```diff
@@ -42,6 +42,7 @@ class Notifier {
       skipTaskbar: true,
       webPreferences: {
         devTools: false,
+	nodeIntegration: true
```
Comment (cahitihac) [inline lib/notifier.js]: Can you please align the new line?

Comment (ahallicks) [reply lib/notifier.js]: Done :-)

**Comment by mariouzae** [comment]
Hi, when this PR will be merged?

thanks

**Comment by bayleedev** [comment]
I can do it now [: have a good day

**Pull request merged**